.github/workflows/ci.yml

```diff
@@ -258,7 +258,7 @@ jobs:
         if: runner.os == 'macOS'
         # NB: rustc doesn't ship libLLVM.so on macOS, so disable proxying (default feature).
         # Remove --rev when LLVM18 is released.
-        run: cargo install bpf-linker --git https://github.com/aya-rs/bpf-linker.git --rev 821f92990074cb7e950e25129dcd55e20424cede --no-default-features
+        run: cargo install --force bpf-linker --git https://github.com/aya-rs/bpf-linker.git --rev 821f92990074cb7e950e25129dcd55e20424cede --no-default-features
```

Comment:
can you add a comment to explain why we're doing this?

Reply:
done